Gemfile

```diff
@@ -24,3 +24,7 @@ end
 platforms :jruby do
   gem 'jruby-openssl'
 end
+
+if RUBY_VERSION >= '3.4'
+  gem 'mutex_m' # TODO: remove this once as-notifications has such a dependency
+end
```

Comment:
`mutex_m` is a missing dependency of `as-notifications`. The dependency should be added to this library.